lib/Graph/Graph.cpp

```diff
@@ -4801,8 +4801,9 @@ BBoxTransformNode *Function::createBBoxTransform(
 
   auto boxOutTy = getParent()->uniqueTypeWithNewShape(
       rois.getType(), {deltasDims[0], deltasDims[1]});
+  // Forcing roiBatchSplitsTy to always be Float.
   auto roiBatchSplitsTy =
-      getParent()->uniqueType(rois.getElementType(), {imInfoDims[0]});
+      getParent()->uniqueType(ElemKind::FloatTy, {imInfoDims[0]});
```

Comment:
@mortzur Can you comment on this change?

Reply:
roiBatchSplits is in fact an integer but it is consumed as float by other ops (i.e. caffe2 NMS). 
This raises a problem that I haven't thought of - how to not convert this specific output to FP16 in case that FP16 global conversion is required.